index.html

```diff
@@ -0,0 +1,238 @@
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w2">
+        <h4>Profesional Art Printing Data</h4>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w3">
+        <h4>Profesional Art Printing Data</h4>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w4">
+        <h4>Profesional Art Printing Data</h4>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w5">
+        <h4>Profesional Art Printing Data</h4>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w6">
+        <h4>Profesional Art Printing Data</h4>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </section>
+    <section class="AboutMe">
+      <div class="about">
+        <h1>About me</h1>
+        <p>
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don't
+          hestiate to contact me.
+        </p> 
+    
+        <button>Get My Resume</button>
+
+      </div>
+    </section>
+    <section class="myskills">
```

Comment:
Great design @Robison-faike123 🤝

- [ ] [Optional] Kindly remove the white space at the bottom of your design. You may try to fix your margin-bottom so as to attain this.
![bottom](https://user-images.githubusercontent.com/63917725/193193405-89f63676-11da-4c14-968b-39aa403ec511.PNG)